### PR TITLE
Develop - Merge to Master

### PR DESCRIPTION
**Pull request by kiritigowda**
* rocAL deprecated
* mv-object readme fix
* Build for GFX941/942 with HIP Version 6.0+

### CODE REVIEW

File: amd_openvx/README.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ To provide a basic set of vision processing functions. This set of functions is
   * **vx_loomsl**: Radeon LOOM stitching library for live 360-degree video applications
   * **vx_nn**: OpenVX neural network module that was built on top of [MIOpen](https://github.com/ROCmSoftwarePlatform/MIOpen)
   * **vx_opencv**: OpenVX module that implemented a mechanism to access OpenCV functionality as OpenVX kernels
-  * **vx_rpp**: OpenVX extension providing an interface to some of the ROCm Performance Primitives ([RPP](https://github.com/ROCm/rpp)) functions. This extension is used to enable [rocAL](../rocAL/README.md) to perform image augmentation.
+  * **vx_rpp**: OpenVX extension providing an interface to some of the ROCm Performance Primitives ([RPP](https://github.com/ROCm/rpp)) functions. This extension is used to enable [rocAL]([../rocAL/README.md](https://github.com/ROCm/rocAL)) to perform image augmentation.
```
Comment (rrawther) [inline amd_openvx/README.md:157]: I thought we still need to build vx_rpp extension in MIVisionX

**Review by rrawther — APPROVED** [review]

**Review by LakshmiKumar23 — CHANGES_REQUESTED** [review]
PLease address review comments. Otherwise looks ok

File: amd_openvx/README.md (hunk 1 above)
Comment (kiritigowda) [reply amd_openvx/README.md:157]: Yes, we are going to build vx_rpp

File: MIVisionX-setup.py [hunk 2]
```diff
@@ -251,88 +223,63 @@
...
         os.system('sudo -v')
         if "centos" in platfromInfo or "redhat" in platfromInfo:
             if "centos-7" in platfromInfo or "redhat-7" in platfromInfo:
                 os.system('sudo '+linuxFlag+' '+linuxSystemInstall+' ' + linuxSystemInstall_check +
-                          ' install kernel-devel libsqlite3x-devel bzip2-devel openssl-devel python3-devel autoconf automake libtool curl make g++ unzip')
+                        ' install kernel-devel libsqlite3x-devel bzip2-devel openssl-devel python3-devel autoconf automake libtool curl make g++ unzip')
             elif "centos-8" in platfromInfo or "redhat" in platfromInfo:
                 os.system('sudo '+linuxFlag+' '+linuxSystemInstall+' ' + linuxSystemInstall_check +
-                          ' install kernel-devel libsqlite3x-devel bzip2-devel openssl-devel python3-devel autoconf automake libtool make gcc-c++ unzip')
+                        ' install kernel-devel libsqlite3x-devel bzip2-devel openssl-devel python3-devel autoconf automake libtool make gcc-c++ unzip')
```
[the remark this answers is not in the document]
Comment (LakshmiKumar23) [reply MIVisionX-setup.py:246]: @kiritigowda That branch still has the same issues. Installs both python-dev and python3-dev

[the remark this answers is not in the document]
Comment (kiritigowda) [reply MIVisionX-setup.py:246]: @LakshmiKumar23 - yes, will create a PR when it's ready.

**Pull request merged**